src/storage/src/source/source_reader_pipeline.rs

```diff
@@ -116,6 +116,8 @@ pub struct RawSourceCreationConfig {
     pub persist_clients: Arc<PersistClientCache>,
     /// Place to share statistics updates with storage state.
     pub source_statistics: StorageStatistics<SourceStatisticsUpdate, SourceStatisticsMetrics>,
+    /// Enables reporting the remap operator's write frontier.
+    pub remap_stashed_upper: Rc<RefCell<Antichain<mz_repr::Timestamp>>>,
```

Comment:
I'm not sure I like bloating this thing further. But I also don't have a good alternative right now. 😅 

If we keep it, could we maybe call it `shared_remap_upper` instead, though?

Reply:
Ah, some comment in `persist_sink` referred to an analogous bit of code as a "stashed frontier"; will adjust this to include your recommended change.